package.json

```diff
@@ -73,6 +73,7 @@
     "eslint": "^5.2.0",
     "eslint-config-prettier": "^2.9.0",
     "eslint-plugin-babel": "^5.1.0",
+    "eslint-plugin-google-camelcase": "^0.0.2",
```

Comment:
Should this be included in the ESLint config?

Reply:
whoops that was an accidental commit